frontend/src/pages/PropertyDetails.jsx

```
@@ -0,0 +1,747 @@
+
+            {/* Full Screen Modal */}
+            {isModalOpen && (
+            <div 
+                className="fixed inset-0 z-50 bg-gray-900/95 flex items-center justify-center p-8"
+                onClick={() => setIsModalOpen(false)}
+                onKeyDown={(e) => {
+                    if (e.key === 'Escape') {
+                        setIsModalOpen(false);
+                    }
+                }}
+                tabIndex={0} // Allows div to capture key events
```

Biome (1.9.4)

[error] The HTML element div is non-interactive. Do not use tabIndex.

Adding non-interactive elements to the keyboard navigation flow can confuse users.
Unsafe fix: Remove the tabIndex attribute.


(lint/a11y/noNoninteractiveTabindex)